pkg/graph/resolvers/trb_request_feedback.go

```diff
@@ -0,0 +1,87 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/storage"
+)
+
+// CreateTRBRequestFeedback creates a TRBRequestFeedback in the database
+func CreateTRBRequestFeedback(ctx context.Context, store *storage.Store, feedback *models.TRBRequestFeedback) (*models.TRBRequestFeedback, error) {
+	feedback.CreatedBy = appcontext.Principal(ctx).ID()
+
+	form, err := store.GetTRBRequestFormByTRBRequestID(ctx, feedback.TRBRequestID)
```

Comment:
If you really wanted, you could do this request concurrently with the `store.GetTRBRequestByID()` call using an errgroup; up to you if you want to implement that level of optimization.